src/index.html

```diff
@@ -11,7 +11,180 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post"
+          onsubmit="onSubmit()"
+    >
```

Comment:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      onsubmit="onSubmit()"
    >
```
Давай в подальшому будемо використовувати таке форматування

Reply:
Добро